lib/LedgerSMB/Scripts/import_csv.pm

```diff
@@ -591,6 +646,11 @@ sub begin_import {
         $template_setup->{$request->{type}}($request);
     }
 
+    $request->{title} = $header_info->{$request->{type}}->{title};
+    $request->{info} = $header_info->{$request->{type}}->{info};
+    $request->{show_batch_title} = 1 if $request->{type} =~ m/multi$/;
+    $request->{show_input} = 1 if ($request->{type} =~ m/multi$/ or $request->{type} eq 'gl');
```

Comment:
This looks like a good criterion, except that `inventory` (but not `inventory_multi`) uses `$request->{transdate}` as well, so that too would need the input to be shown.

Reply:
Maybe a more flexible solution is to add the "show_input" boolean to the "$header_info" hashes and make them 0/1 or ''/1 explicitly in those. That makes the process of showing them even more transparent.

Reply:
> This looks like a good criterion, except that `inventory` (but not `inventory_multi`) uses `$request->{transdate}` as well, so that too would need the input to be shown.

Inventory(multi) import use different html template file. So we dont need to consider for inventory in this case.

Previously, I add show_input in header_info hash. But when I realize only multi... and gl use these fields, I move to the current approach.